app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -49,7 +51,13 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
+    /**
+     * Used to fill the listview of newly discovered Bluetooth devices
+     */
     private ArrayAdapter<String> newDevicesArrayAdapter;
+    /**
+     * Used to fill the listview of paired Bluetooth devices
```

Comment:
Yeah, but Javadoc is always between /** **/, and this is Javadoc, no ordinary comment